File: versioncontrol/src/main/java/ir/ayantech/versioncontrol/model/VCRequestModel.java
package ir.ayantech.versioncontrol.model;

import com.google.gson.Gson;

/**
 * Created by Administrator on 6/10/2017.
 */

public class VCRequestModel {
    private VCInputModel Parameters;

    public VCRequestModel(VCInputModel Parameters) {
        this.Parameters = Parameters;
    }

    public VCRequestModel() {
    }

    public String toString() {
        return new Gson().toJson(this);
    }
}

File: versioncontrol/src/main/java/ir/ayantech/versioncontrol/model/VCRequestModel.kt
package ir.ayantech.versioncontrol.model

import com.google.gson.Gson
import com.google.gson.annotations.SerializedName

open class VCRequestModel(
    @SerializedName("Parameters")
    var parameters: VCInputModel? = null
) {
    override fun toString(): String {
        return Gson().toJson(this)
    }
}
